docs/site_specific_config/rocm.md

```diff
@@ -320,36 +299,134 @@ graph LR;
-* rocFFT: Fast Fourier Transform implementation
-    * [Github](https://github.com/ROCm/rocFFT)
-* rocRAND: Random number generator library
-    * [Github](https://github.com/ROCm/rocRAND)
-* rocSPARSE: Sparse matrix routines
-    * [Github](https://github.com/ROCm/rocSPARSE)
+* rocFFT: Fast Fourier Transform implementation ([Github](https://github.com/ROCm/rocFFT))
+* rocRAND: Random number generator library ([Github](https://github.com/ROCm/rocRAND))
+* rocBLAS: Basic Linear Algebra Subprograms implementation ([Github](https://github.com/ROCm/rocBLAS))
+* rocSPARSE: Sparse matrix routines ([Github](https://github.com/ROCm/rocSPARSE))
+* rocSOLVER: Linear algebra solver library ([Github](https://github.com/ROCm/rocSOLVER))
+* hipBLASLt: General matrix-matrix operations, extends beyond BLAS ([Github](https://github.com/ROCm/hipBLASLt))
+* hipSPARSELt: Marshalling library and ROCm version of cuSPARSELt ([Github](https://github.com/ROCm/hipSPARSELt))
 
 ### ML/DL Frameworks
 
-* MIOpen: Deep learning primitives library
-    * [Github](https://github.com/ROCm/MIOpen)
-* ROCm TensorFlow: TensorFlow support for AMD GPUs
-    * [Github](https://github.com/ROCm/tensorflow-upstream)
-* ROCm PyTorch: PyTorch support for AMD GPUs
-    * [Github](https://github.com/ROCm/pytorch)
-* RCCL: Communication library for multi-GPU/multi-node training
-    * [Github](https://github.com/ROCm/rccl)
+* MIOpen: Deep learning primitives library ([Github](https://github.com/ROCm/MIOpen))
+* ROCm PyTorch: PyTorch support for AMD GPUs ([Github](https://github.com/ROCm/pytorch))
+* ROCm TensorFlow: TensorFlow support for AMD GPUs ([Github](https://github.com/ROCm/tensorflow-upstream))
 
 ### Communication Libraries
 
-* ROCm Communication Collectives Library (RCCL): Optimized collective operations
-    * [Github](https://github.com/ROCm/rccl)
-* UCX: Unified Communication X support
-    * [Github](https://github.com/openucx/ucx)
-* ROCm MPI: Message Passing Interface integration
-    * [Github](https://github.com/ROCm/rocm_smi_lib)
+* RCCL: Communication library for multi-GPU/multi-node training ([Github](https://github.com/ROCm/rccl))
+
+### Marshalling Libraries
+
+* hipFFT ([Github](https://github.com/ROCm/hipFFT))
+* hipRAND ([Github](https://github.com/ROCm/hipRAND))
+* hipBLAS ([Github](https://github.com/ROCm/hipBLAS))
+* hipSPARSE ([Github](https://github.com/ROCm/hipSPARSE))
+* hipSOLVER ([Github](https://github.com/ROCm/hipSOLVER))
+
+## Dependencies Graph
+
+```mermaid
+graph LR;
+    subgraph Core Components
+        driver[AMDGPU Driver]
+        runtime[ROCm Platform Runtime]
+        llvm[ROCm LLVM Compiler]
+        hip[HIP]
+        amdsmi[AMD SMI]
+        rocmsmi[ROCm SMI]
+        rocmcmake[ROCm CMake]
+        rocminfo[ROCm Info]
+        rocmexamples[ROCm Examples]
+    end
+    subgraph Programming Models
+        openmp[OpenMP Support]
+        opencl[OpenCL Support]
+    end
+    subgraph Developer Tools
```

Comment:
Is there any way to make this block vertical, so whole rendered graph is not as wide?